src/content/peerconnection/create-offer/js/main.js

```diff
@@ -53,10 +52,11 @@ function createOffer() {
     voiceActivityDetection: vadInput.checked
   };
 
-  pc.createOffer(offerOptions)
-    .then(desc => {
-      pc.setLocalDescription(desc);
-      outputTextarea.value = desc.sdp;
-    })
-    .catch(error => outputTextarea.value = `Failed to createOffer: ${error}`);
+  try {
+    const desc = await pc.createOffer(offerOptions);
+    pc.setLocalDescription(desc);
```

Comment:
setLocalDescription() returns a promise. Shouldn't you await it as well or does it not matter for this sample?